news/bugfix-3144.md

```diff
@@ -0,0 +1,2 @@
+`python`: Now we store key-values in UTF-8 `str` instead of `byte`,
```

Comment:
Python 3 strings are not UTF-8, let's just mention `str` or "Python strings" without the actual encoding.

Reply:
Thanks, fixed